### ISSUE TITLE
[WebToolsE2E][Aspire] Under 'Metrics' of dashboard, some contents cannot catch up with the theme changes, hard to read and need to refresh the page.

### ISSUE THREAD

**Issue by v-cazhang** [dotnet/aspire#401]
**INSTALL STEPS**

1. Clean machine: Win11 x64 22h2 ENU
2. Install VS 17.9 P1 from build main-34210.225
3. Install latest 8.0 SDK: 8.0.100-rtm.23506.1
4. Apply NuGet Feeds
5. Use PAT to access private feed.
6. Install Aspire workload

**REPRO STEPS**

1. In the VS, create a new project > Aspire Starter Application > give a name > .NET 8.0 > create.
2. F5/Ctrl F5
3. On the dashboard, navigate to 'Metrics', expand 'Select service..' and select 'webfrontend', click any item.
4. Click the settings button, change the theme.

**ACTUAL**
Some contents like 'chart' cannot catch up with the theme change.
change 'Dark' to 'Light':
![image](https://github.com/dotnet/aspire/assets/82028229/1acf3078-0628-4064-a5af-e41b6a368410)
change 'Light' to 'Dark':
![image](https://github.com/dotnet/aspire/assets/82028229/20847639-a3f9-4e08-93e5-f48963815b78)

**EXPECTED:**
Contents can change immediately when changing the theme.
